PWGCF/Tasks/correlations.cxx

```diff
@@ -417,11 +475,16 @@ struct CorrelationTask {
         }
 
         // last param is the weight
-        if (cfgMassAxis) {
+        if (cfgMassAxis && !cfgDDBARana) {
```

Comment:
`&& !cfgDDBARana` not needed if you move your new branch below before this one and do `else if` here